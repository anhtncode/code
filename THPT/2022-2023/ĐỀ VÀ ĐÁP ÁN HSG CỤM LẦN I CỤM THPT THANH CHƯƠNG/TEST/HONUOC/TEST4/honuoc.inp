23 15
6181 90
2385 76
4306 80
3236 39
7503 47
8509 18
8381 86
5571 37
3617 42
6905 59
4125 51
4074 57
9319 44
6650 8
3290 25
7792 80
8654 22
3952 92
4258 31
4071 30
3850 2
7376 77
4694 89
