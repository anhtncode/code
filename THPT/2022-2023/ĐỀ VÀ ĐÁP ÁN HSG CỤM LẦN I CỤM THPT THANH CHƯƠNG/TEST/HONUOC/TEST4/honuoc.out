128699
127517
126335
125153
123971
122789
121607
120425
119243
118061
116879
115697
114515
113333
112151
110969
